Hydration_freenrg/energy_results/results.csv

```diff
@@ -0,0 +1,17 @@
+,mol0,mol1,free_energy,error,successful_reps,date
+0,2-cyclopentanylindole,7-cyclopentanylindole,-13.046526666666663,0.0,3,2023-06-05
+0,2-Methylfuran,Methane,2.7067900000000003,0.22703460088717753,3,2023-06-05
+0,2-Methylindole,Methane,8.441430000000002,0.38098251928402177,3,2023-06-05
+0,7-cyclopentanylindole,2-cyclopentanylindole,3.8650366666666662,0.0,3,2023-06-05
+0,Ethane,Methane,-0.019610000000001016,0.13923329343228222,3,2023-06-05
+0,Ethane,Methanol,-6.310936666666666,0.058159693946925134,3,2023-06-05
+0,Methane,2-Methylfuran,-2.7191600000000005,0.18426556922007972,2,2023-06-05
+0,Methane,2-Methylindole,-8.646020000000002,0.2861011359641901,2,2023-06-05
+0,Methane,Ethane,0.012183333333333582,0.1424585553766428,3,2023-06-05
+0,Methane,Methanol,-5.889689999999999,0.10670576366813557,3,2023-06-05
+0,Methane,Neopentane,,0.0,0,2023-06-05
+0,Methane,Toluene,-3.014869999999999,0.17036507858126326,2,2023-06-05
+0,Methanol,Ethane,6.333780000000001,0.05828790612125297,3,2023-06-05
+0,Methanol,Methane,6.019496666666666,0.11113752741536047,3,2023-06-05
+0,Neopentane,Methane,-0.3749933333333331,0.1807107080391198,3,2023-06-05
+0,Toluene,Methane,3.0373866666666665,0.19069040353410552,3,2023-06-05
```

Comment:
Nice. I like that GitHub automatically renders this as a table so it's easy to view on the web. For next time I wonder if it would be useful to have a corresponding `environments` directory with dated conda environment files to match the time-stamp in the results table, e.g. `2023-06-05.yml`. This could be generated within one of your scripts using, e.g;

```sh
conda env export > environments/$(date +"%Y-%d-%m").yml
```

(Obviously the environment needs to be active first.)

Reply:
I suppose the point in the process at which this file is generated is not too important (assuming that `biosimspace`  isn't updated between, for example, running the Minimisation/Equilibration and running the analysis), although it probably makes the most sense to generate it at the same time as the min/eq as this has the most calls to `biosimspace` functionality.

Reply:
The environment will be the same throughout so it shouldn't matter where you do it. I only think it's useful since some of our dependencies aren't strictly pinned, so it's possible that someone wishing to re-run could end up with a slightly different version of a dependency, which could give different results, e.g. a different version of RDKit giving a different mapping, or a slightly different version of OpenMM causing issues. The version of Sire and BioSimSpace alone aren't enough to fully specify the environment.

We'd probably need to think about logging versions of external engines, such as GROMACS, but can do that when we add support for those. (These might be outside of the conda environment.)